net-firewall/ferm/ferm-2.3.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils systemd
```

Comment:
einstalldocs is an EAPI provided helper. See https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-145034r25. you can safely remove eutils if that's the only assumed use of it.